<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentOperations</class>
 <widget class="QWidget" name="AgentOperations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="agentNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Agent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateInvitationButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generate Invitation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterInvitationButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Invitation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkCredentialButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Check Credentials</string>
   </property>
  </widget>
  <widget class="QPushButton" name="publishSchemaButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Publish Crednetial Schema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="issueCredentialButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Issue Credential</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serviceRequestButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Request Service</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="responseTextBrowser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9.16364pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.16364pt;&quot;&gt;Waiting for response.....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="agent_name_label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>16</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Response</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
